### PR TITLE
Add single bead thermal history parameters

### PR DESCRIPTION
**Pull request by safeerehman**
- Update `single_bead.py` to include the parameters
- Update 'test_single_bead.py` for `__repr__` change
- Update `ci_cd.yaml` to trigger runs

@Todo 
- Add more tests to verify the new arg values for the `SingleBeadInput` class constructor.
- Update the example to show the thermal history for single bead

### CODE REVIEW

**Comment by pkrull-ansys** [comment]
@safeerehman Update the example for single bead to show the thermal history.

File: src/ansys/additive/core/single_bead.py [hunk 1]
```diff
@@ -120,6 +133,30 @@ def bead_length(self, value):
         self.__validate_range(value, self.MIN_BEAD_LENGTH, self.MAX_BEAD_LENGTH, "bead_length")
         self._bead_length = value
 
+    @property
+    def output_thermal_history(self) -> bool:
+        """Flag to output the thermal history of the simulation."""
```
Comment (PipKat) [inline src/ansys/additive/core/single_bead.py:138]:
```suggestion
        """Flag indicating if the thermal history of the simulation is to be outputted."""
```

File: src/ansys/additive/core/single_bead.py [hunk 2]
```diff
@@ -120,6 +133,30 @@ def bead_length(self, value):
...
+        return self._output_thermal_history
+
+    @output_thermal_history.setter
+    def output_thermal_history(self, value: bool):
+        self._output_thermal_history = value
+
+    @property
+    def thermal_history_interval(self) -> int:
+        """Interval to output thermal history data."""
```
Comment (PipKat) [inline src/ansys/additive/core/single_bead.py:147]:
```suggestion
        """Interval for outputting thermal history data."""
```

**Comment by pkrull-ansys** [comment]
@safeerehman Is this superseded by #363?

**Comment by safeerehman** [comment]
These changes will be a part of #363

**Pull request closed without merging**